modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -492,17 +498,65 @@ void cv::gimpl::ie::GIEExecutable::run(std::vector<InObj>  &&input_objs,
 namespace cv {
 namespace gimpl {
 namespace ie {
+inline void configureReshapeByInputImage(const IE::InputInfo::Ptr& ii,
+                                         const IEUnit& uu,
+                                         const std::vector<int>& image_size) {
+    // Name for current layer
+    const auto layer_name = ii->name();
+    // Finding name in params
+    const auto name_pos_in_params =
+        ade::util::find(uu.params.layer_names_to_reshape, layer_name);
+    // Finding name in reshape table
+    const auto name_pos_in_table =
+        ade::util::find_if(uu.input_reshape_table,
```

Comment:
No, it's a worst approach to find something in `std::map`

Reply:
Fixed.